system/core/pages.php

```diff
@@ -47,17 +47,57 @@ abstract class HyphaSystemPage extends HyphaPage {
 		abstract class for a page backed by datatype and xml file.
 	*/
 	abstract class HyphaDatatypePage extends HyphaPage {
+		const INDEX_TABLE_COLUMNS_TITLE = 'title';
+		const INDEX_TABLE_COLUMNS_AUTHOR = 'author';
+		const INDEX_TABLE_COLUMNS_DATE = 'date';
+		const INDEX_TABLE_COLUMNS_STATUS = 'status';
```

Comment:
Would it make sense to add a prefix to these translation identifiers to indicate that they will be used as column headers?

Reply:
Good point! Will do that!

Reply:
Also, I think this is the first time we're using constants as translation string ids (instead of passing them to `__()` literally). If we ever want to add some automatic translation checking, which I can imagine works by scanning the code for calls to `__()` and picking out the argument, this might make that more complicated.